src/schemas/Program/index.js

```diff
@@ -292,45 +58,39 @@ const resolvers = {
 		cancerTypes: async (constants, args, context, info) => {
 			const { egoToken } = context;
 			const response = await programService.listCancers(egoToken);
-			return get(response, 'cancers', [])
-				.map((cancerType) => cancerType.name.value)
-				.sort();
+			return response || null;
 		},
 		primarySites: async (constants, args, context, info) => {
 			const { egoToken } = context;
 			const response = await programService.listPrimarySites(egoToken);
-			return get(response, 'primary_sites', [])
-				.map((site) => site.name.value)
-				.sort();
+			return response || null;
 		},
 		institutions: async (constants, args, context, info) => {
 			const { egoToken } = context;
 			const response = await programService.listInstitutions(egoToken);
-			return get(response, 'institutions', [])
-				.map((institution) => institution.name.value)
-				.sort();
+			return response || null;
 		},
 		regions: async (constants, args, context, info) => {
 			const { egoToken } = context;
 			const response = await programService.listRegions(egoToken);
-			return get(response, 'regions', [])
-				.map((region) => region.name.value)
-				.sort();
+			return response || null;
 		},
 		countries: async (constants, args, context, info) => {
 			const { egoToken } = context;
 			const response = await programService.listCountries(egoToken);
-			return get(response, 'countries', [])
-				.map((country) => country.name.value)
-				.sort();
+			return response || null;
 		},
 	},
 	Program: {
 		users: async (program, args, context, info) => {
 			const { egoToken } = context;
-			const response = await programService.listUsers(program.shortName, egoToken);
-			const users = response ? get(response, 'userDetails', []).map(convertGrpcUserToGql) : null;
-			return users;
+			const users = await programService.listUsers(egoToken, program.shortName);
+			return users || null;
+		},
```

Comment:
Investigating bug 'cannot return null for non-nullable field ProgramUser.email'

Reply:
OK as far as I can tell, several Users exist with `role = null`
This did not match the Type def, so the entire User was returned as `null`, likely causing the above `ProgramUser.email` error (I could not find any records where email was null)

User objects with `role = null` also caused problems for Puneet; she was unable to open an email invite link
Some of the User objects where `role = null` also have `"inviteStatus": "EXPIRED"`, so it might make sense in these cases

For now I am making the field nullable to allow the Query to succeed
But this needs a follow up ticket as it is also the source of bugs, and we should make sure User role is only `null` in acceptable cases